src/modules/edc-demo/components/introduction/introduction.component.html

```diff
@@ -1,7 +1,7 @@
 <div id="wrapper">
     <mat-card>
-    <h1>Eclipse Dataspace Connector</h1>
-    <div>
+      <mat-card-header><h1>Eclipse Dataspace Connector</h1></mat-card-header>
+    <mat-card-content>
```

Comment:
```suggestion
      <mat-card-header>
        <mat-card-title>Eclipse Dataspace Connector</mat-card-title>
      </mat-card-header>
      <mat-card-content>
```

Using `mat-card-title` to follow material design.